lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -39,113 +39,226 @@
     %>
   % else:
     <% mode_class = '' %>
-   % endif
-
-  <article class="course${mode_class}">
-
-    <%
-      course_target = reverse('info', args=[course.id.to_deprecated_string()])
-    %>
+  % endif
 
-    % if show_courseware_link:
-      % if not is_course_blocked:
-        <a href="${course_target}" class="cover">
-        <img src="${course_image_url(course)}" alt="${_('{course_number} {course_name} Home Page').format(course_number=course.number, course_name=course.display_name_with_default) |h}" />
-      </a>
+<article class="course${mode_class}">
+  <% course_target = reverse('info', args=[course.id.to_deprecated_string()]) %>
```

Comment:
Nitpick: `to_deprecated_string()` is deprecated :)  Could you use `unicode(course.id)` instead?